README.md

```diff
@@ -93,6 +74,41 @@ Each runs their respective process configured as either a `-controller` or an `-
   The on-host process responsible for publishing update metadata and executing
   update activities.
 
+## Getting Started
+
+### Label nodes
+
+To start Bottlerocket updater operator agent on nodes, you will need to add `bottlerocket.aws/updater-interface-version` label.
+We recommend using `update-interface-version` 2.0.0 for Bottlerocket OS version >=v0.4.1 which uses Bottlerocket's [update API](https://github.com/bottlerocket-os/bottlerocket/blob/develop/sources/updater/README.md#update-api) to dispatch updates.
+
+With `kubectl` configured for the desired cluster, you can use the below command to get all nodes
+
+```sh
+kubectl get nodes
+```
+Make a note of all the node names that you would like Bottlerocket update operator to manage.
+
+Next, add `updater-interface-version` label to the nodes. 
+For each node, use this command to add `updater-interface-version` label. 
+Make sure to change NODE_NAME with the name collected from the previous command.
+
+```sh
+kubectl label node NODE_NAME bottlerocket.aws/updater-interface-version=2.0.0
+```
+
+If all nodes in the cluster are running Bottlerocket and require the same `updater-interface-version`, you can label all at the same time by running this
+```sh
+kubectl label node $(kubectl get nodes -o jsonpath='{.items[*].metadata.name}') bottlerocket.aws/updater-interface-version=2.0.0
```

Comment:
I'm being picky here, but this will only work in bash-like shells. I use `fish` and I can't just copy/paste this command. I'm posting this just to let people know that not all the commands that start a subshell will work for all the users.